README.md

```diff
@@ -41,6 +41,16 @@ must be applied via [patches](patches). See [below](#patches) for more info on p
 
 ## Updating an External Component to a Newer Version
 
+When updating a component that is used in both Arcade and another target repository, follow these steps:
+
+1. **Add the updated component**: Include the component with the updated version as a new submodule in source-build-externals.
+2. **Update the target repository**: Allow the updated version to flow to the target repository and update the target repository to use this new version.
+3. **Build and release**: Perform a build and release the artifacts to update the n-1 (previous) artifacts for Arcade.
+4. **Update Arcade**: Update Arcade to use the new version of the component.
```

Comment:
Per my previous comment this should not be hard coded to arcade.  It would be nice to have clear terms to differentiate this type of repo reference versus `target repository` used earlier.  I don't know if referring to SBRP and Arcade as a bootstrap/toolset repos would help.

Reply:
I added references to `toolset repository` instead of `Arcade` in [this commit](https://github.com/dotnet/source-build-externals/pull/280/commits/3c856a4815ff8133ec423b50ef6975d97aee6f2c).